src/api/spec/models/attrib_spec.rb

```diff
@@ -1,19 +1,106 @@
 require 'rails_helper'
 
-RSpec.describe Attrib do
+RSpec.describe Attrib, :type => :model do
   let(:attribute) { create(:attrib, project: create(:project)) }
   let(:project) { create(:project) }
   let(:package) { create(:package) }
 
+  context "#fullname" do
```

Comment:
I think this should be `describe` not `context`? There are several places where context is used incorrectly in this spec. See these links about the difference between describe/context:
http://www.betterspecs.org/#describe
http://www.betterspecs.org/#contexts